compiler/quilt/tools/command.py

```diff
@@ -131,7 +131,12 @@ def _save_auth(cfg):
 
 def get_registry_url(team):
```

Comment:
What happens when a team is configured, but the user does something like `quilt install foo/bar` without a team prefix?

I think you may need to modify `url = cfg.get('registry_url', '')` below, and make sure a team URL is not returned if team is `None`.